docs/src/main/asciidoc/doc-reference.adoc

```diff
@@ -329,6 +329,7 @@ Quarkus documentation is grouped into the following categories.
 | integration | Support for integration extensions (Camel)
 | messaging | Integrations with messaging systems like Kafka, AMQP, or RabbitMQ.
 | miscellaneous | Miscellaneous
+| native | Information about building native applications with Quarkus
```

Comment:
@ebullient - Is the 'native' category active and usable on the index page yet?  If not, I will remove from here. Thanks.